bindings.lisp

```diff
@@ -181,6 +181,7 @@ is a tile group.")
   (kbd "f")       "fselect"
   (kbd "F")       "curframe"
   (kbd "-")       "fclear"
+  (kbd "q")       "quit"
```

Comment:
I'm not sure I'm convinced binding quit by default is a good idea.

Reply:
I understand this concern, which is why it also is set to prompt by default. Otherwise a user poking around in StumpWM could accidentally close everything and have no idea what happened.

Reply:
I also find myself having to quit StumpWM fairly often, because stuff breaks and configuration involves testing things.